### PR TITLE
Added a reminder about Go to member

### PR DESCRIPTION
**Pull request by phistuck**

### CODE REVIEW

File: src/content/en/updates/posts/2016/04/devtools-digest-command-menu.md [hunk 1]
```diff
@@ -37,6 +37,11 @@ is in focus!) to bring up the Command Menu, then type to filter and hit
 
 The new command menu is a super quick way to navigate and discover new settings and actions across DevTools.
 
+Looking for the good old "Go to member" dialog that was previously also
+associated with that shortcut? It is still there, just hit
```
Comment (jpmedley) [inline src/content/en/updates/posts/2016/04/devtools-digest-command-menu.md]: Should be 'hidden', not 'hit' (or 'hid').

**Comment by jpmedley** [comment]
@phistuck Can you please fix the typo I found.

File: src/content/en/updates/posts/2016/04/devtools-digest-command-menu.md (hunk 1 above)
Comment (phistuck) [inline src/content/en/updates/posts/2016/04/devtools-digest-command-menu.md]: Mmm... no, it should be "hit". "just hit (shortcut keys) from now on".

Comment (phistuck) [inline src/content/en/updates/posts/2016/04/devtools-digest-command-menu.md]: Just like it says above -
```
 Hit <kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>
 (or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>) anywhere (even when the page 
```

**Comment by phistuck** [comment]
Err.... no. :) See my reply to the comment.

Note - I get e-mail notifications for the note you add, you can avoid asking me to take care of those in a comment as well. :)

**Comment by jpmedley** [comment]
Thought I was looking at the end of a sentence. Sorry.

**Pull request merged**